conda/recipe/build.sh

```diff
@@ -158,7 +158,16 @@ export LD_LIBRARY_PATH="$PREFIX/libjpeg-turbo/lib:$PREFIX/lib:$LD_LIBRARY_PATH"
 DALI_PATH=$($PYTHON -c 'import nvidia.dali as dali; import os; print(os.path.dirname(dali.__file__))')
 echo "DALI_PATH is ${DALI_PATH}"
 pushd $SRC_DIR/dali_tf_plugin/
-source ./build_dali_tf.sh $DALI_PATH/plugin/libdali_tf_current.so
+mkdir -p prebuilt/plugin
+source ./build_dali_tf.sh prebuilt/plugin/libdali_tf_current.so
```

Comment:
I think you can skip that, as it is going to be build during `pip install`

Reply:
Done